src/components/Modal/ModalWrapper.tsx

```diff
@@ -121,46 +118,74 @@ const ModalWrapper: React.FC<ModalWrapperProps> = ({
   className,
   style,
   padding,
+  children,
 }) => {
   /**
    * Renders the appopriate variant based on the availability of a
    * panel prop and its position.
    * @param modalProps
    */
-  const ModalProxy = (modalProps: {
-    maxContentHeight: string
-    minContentHeight: string
-    header: ReactNode
-    footer: ReactNode
-    body: ReactNode
-  }) => {
+  const ModalProxy = (maxContentHeight: string, minContentHeight: string) => {
     if (!panel) {
-      return <Modal {...modalProps} />
+      return (
+        <Modal
+          padding={padding}
+          maxContentHeight={maxContentHeight}
+          minContentHeight={minContentHeight}>
+          {children}
+        </Modal>
+      )
     }
-    switch (panel.position) {
+    switch (panel) {
       case "left":
-        return <LeftPanelModal {...modalProps} panelContent={panel.content} />
+        return (
+          <LeftPanelModal
```

Comment:
Since we are having identical props and children, lets only return the component, `return LeftPanelModal`
and then call it where we received the returned value.